modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/Common.cs

```diff
@@ -7,16 +7,18 @@ namespace Godot.SourceGenerators
 {
     public static class Common
     {
+        private const string helpLinkFormat = "https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/diagnostics/{0}.html";
```

Comment:
We should probably use the specific Godot version here instead of _stable_.

Reply:
It's available at build time in `methods.py`, or at compile time as a `VERSION_DOCS_URL` define in C++.